sql/schema/v1-core.sql

```diff
@@ -327,6 +327,7 @@ CREATE TYPE v1_task_event_type AS ENUM (
 -- CreateTable
 CREATE TABLE v1_task_event (
     id bigint GENERATED ALWAYS AS IDENTITY,
+    inserted_at timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Seeing `NOT NULL` here which doesn't seem to match the migration?

Reply:
ah whoops, thanks, I keep doing this 🤦